terraform/gcp/variables.tf

```diff
@@ -50,9 +50,8 @@ variable "k8s_version_prefixes" {
   # channel, so we may want to remove or add minor versions here over time.
   #
   default = [
-    "1.27.",
-    "1.28.",
```

Comment:
All our GKE clusters are at k8s 1.29, so we don't care about 1.27 and 1.28 versions.